heat_infoblox/object_manipulator.py

```diff
@@ -36,32 +36,68 @@ def get_member(self, member_name, return_fields=None, extattrs=None):
             'member', obj, return_fields, extattrs
         )
 
-    def create_member(self, name=None, platform='VNIOS',
-                      mgmt=None, lan1={}, lan2=None,
-                      nat_ip=None):
+    def create_member(self, name=None, platform='VNIOS', ha_pair=False,
+                      vip=None, mgmt=None, lan2=None, nat_ip=None,
+                      use_v4_vrrp=False,
+                      node1_ha=None, node2_ha=None,
+                      node1_lan1=None, node2_lan1=None,
+                      node2_mgmt=None, vrid=None, lan2_vrid=None):
```

Comment:
these are a lot of parameters for a method.
I would think that creating an entity object to encapsulate them and pass it to this method would be better.